libs/etterlatte-ktor/src/main/kotlin/no/nav/etterlatte/libs/ktorobo/DownstreamResourceClass.kt

```diff
@@ -221,7 +221,14 @@ class DownstreamResourceClient(
             .fold(
                 onSuccess = { response ->
                     when {
-                        response.status.isSuccess() -> Ok(response.body())
+                        response.status.isSuccess() -> {
+                            if (response.harContentType(ContentType.Application.Json)) {
+                                Ok(response.body<JsonNode>())
+                            } else {
+                                logger.info("Mottok content-type: ${response.contentType()} som ikke var JSON")
+                                Ok(response.status)
+                            }
+                        }
```

Comment:
Mente å kommentere på dette. Fare for at vi mister responsobjekt/tekst ved å gjøre dette